kursinfo-api-v2.md

```diff
@@ -0,0 +1,21 @@
+## Current
+
+| Method | name          | endpoint                       | comment                                                 |
+| ------ | ------------- | ------------------------------ | ------------------------------------------------------- |
+| GET    | imageStatus   | /v1/images/status              | Get all image names uploaded by user                    |
+| GET    | sellingInfo   | /v1/sellingInfo/{courseCode}   | Returns entire object                                   |
+| POST   | sellingInfo   | /v1/sellingInfo/{courseCode}   | Saves an object in the database (but NOT courseWebLink) |
+| POST   | imageInfo     | /v1/imageInfo/{courseCode}     | Saves random image                                      |
+| POST   | courseWebLink | /v1/courseWebLink/{courseCode} | Saves a boolean flag isCourseWebLink in the database    |
+
+## Proposal
+
+| Method | name          | endpoint                       | comment                                                 | status     |
+| ------ | ------------- | ------------------------------ | ------------------------------------------------------- | ---------- |
+| GET    | imageStatus   | /v1/images/status              | Get all image names uploaded by user                    |            |
+| POST   | imageInfo     | /v1/imageInfo/{courseCode}     | Saves random image                                      |            |
+| GET    | sellingInfo   | /v1/sellingInfo/{courseCode}   | Returns entire object                                   | deprecated |
+| POST   | sellingInfo   | /v1/sellingInfo/{courseCode}   | Saves an object in the database (but NOT courseWebLink) | deprecated |
+| POST   | courseWebLink | /v1/courseWebLink/{courseCode} | Saves a boolean flag isCourseWebLink in the database    | deprecated |
+| PUT    | courseInfo    | /v1/courseInfo/{courseCode}    | Creates/replaces the courseInfo object in the DB        | new        |
+| GET    | courseInfo    | /v1/courseInfo/{courseCode}    | Creates/replaces the courseInfo object in the DB        | new        |
```

Comment:
Change "Creates/replaces the courseInfo object" to "Get the courseInfo object" for GET. 
And add POST courseInfo.

Reply:
Right. I actually aimed to remove that file because i noticed it maybe is just something easily forgotten and duplicates Swagger. What do you think, does it fill a need, @karlandindrakryggen @vsdkth ?

Reply:
Sound good to remove it instead.

Reply:
Fixed 3f6d801